new/LIT7444PPLofham.xml

```diff
@@ -0,0 +1,95 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7444PPLofham" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">እንዘ፡ ይብል፡ አስማተ፡ ሎፍሐም፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾǝnza yǝbǝl ʾasmāta Lofḥam</title>
```

Comment:
Does it start without a capital because the beginning is missing? This makes sense, but I think it might look confusing at first glance. Maybe it might be good to also add ... to the transcription